src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data_clear_experience_configs.py

```diff
@@ -0,0 +1,186 @@
+        """
+        op.drop_constraint(
+            "privacyexperienceconfighistory_experience_config_id_fkey",
+            "privacyexperienceconfighistory",
+            type_="foreignkey",
+        )
+
+    def delete_experiences(bind):
+        """
+        Delete all PrivacyExperience records.
+
+        These records will be effectively re-created as a result of the "out of the box" creation
+        of 'new-style' PrivacyExperienceConfig records post-migration.
+        """
+        bind.execute(
+            text(
+                """
+                DELETE FROM privacyexperience;
+                """
+            )
+        )
+
+    def delete_experience_configs(bind):
+        """
+        Delete all PrivacyExperienceConfig records.
+
+        These records will be effectively re-created via the server's "out of the box" creation
+        of the updated PrivacyExperienceConfig records, post-migration.
+        """
+        bind.execute(
+            text(
+                """
+                DELETE FROM privacyexperienceconfig;
+                """
+            )
+        )
+
+    drop_fk_constraint(bind)
+    delete_experiences(bind)
+    delete_experience_configs(bind)
+
+
+def migrate_notices(bind):
+    """
+    Migrates existing PrivacyNotice records to the new, translation-based data model.
+
+    Notice keys are not impacted. Most existing notice data is moved onto a linked NoticeTranslation record,
+    which is assumed to be english-language as a default. If the existing notice text/content is _not_ english-language,
+    then users will need to *manually* adjust the language on the created translation record.
+
+    The existing PrivacyNoticeHistory records are adjusted to point to the associated NoticeTranslation record,
+    as is expected in the new data model.
+    """
+
+    get_notices_query = text(
+        """SELECT id from privacynotice;
+         """
+    )
+
+    create_notice_translation_query = text(
+        """
+        INSERT INTO noticetranslation (id, privacy_notice_id, language, title, description, created_at, updated_at)
+        SELECT :record_id, :original_notice_id, :language, name, description, created_at, updated_at
+        FROM privacynotice
+        WHERE id = :original_notice_id
+        """
+    )
+    for res in bind.execute(get_notices_query):
+        # Create a Notice Translation for each notice, setting to english as a default
+        bind.execute(
+            create_notice_translation_query,
+            {
+                "record_id": generate_record_id("pri"),
+                "language": "en",
+                "original_notice_id": res["id"],
+            },
+        )
+
+    # Add a FK from the Notice History back to the Notice Translation
+    link_notice_history_to_translation_id = text(
+        """
+        UPDATE privacynoticehistory
+        SET translation_id = noticetranslation.id, title = privacynoticehistory.name
```

Comment:
Should we populate the language too? knowing it might be wrong

Reply:
oh, yeah i think that would make sense - forgot we had that column on this table!